extensions/amp-analytics/amp-analytics.md

```diff
@@ -63,6 +63,7 @@ when the document is first loaded, and each time an `<a>` tag is clicked:
     - `chartbeat`: Adds support for Chartbeat. More details for adding Chartbeat support can be found at [support.chartbeat.com](http://support.chartbeat.com/docs/).
     - `comscore`: Adds support for comScore Unified Digital Measurement™ pageview analytics. Requires defining *var* `c2` with comScore-provided *c2 id*.
     - `googleanalytics`: Adds support for Google Analytics. More details for adding Google Analytics support can be found at [developers.google.com](https://developers.google.com/analytics/devguides/collection/amp-analytics/).
+    - `krux`: Adds support for Krux.  Configuration details can be found at [help.krux.com](http://help.krux.com/).
```

Comment:
Ok but it will still be behind a login as our documentation is restricted to account holders.